### DISCUSSION TITLE
Automating download

### DISCUSSION THREAD

**Discussion by lisim** [danielfernau/unifi-protect-video-downloader#209] [Q&A]
Hi Team

I can't find a way to download on a regular with out me manually running the command since i need to specify the date range

Is there a way like with rcon I can set it to download the prev days footage?

**Comment by danielfernau** [comment]
Hey there,

have a look at 
https://github.com/danielfernau/unifi-protect-video-downloader/issues/69#issuecomment-966663702 and https://github.com/danielfernau/unifi-protect-video-downloader/issues/59#issuecomment-897949011
where inline usage of the `date` command for the `--start` and `--end` parameters is explained.
This can also be used for configuring cron jobs.

In case this doesn't cover your use case just let me know and I'll see what I can do!
